.gitignore

```diff
@@ -1,4 +1,6 @@
 /dist/
 /stack.yaml
+ghcid.txt
```

Comment:
How come you are ignoring `ghcid.txt`? What are you putting in there?

Reply:
Only because i didn't see it I the original repo and figured you didn't want it

Reply:
I don't. I just wondered why you had it? Are you using VS Code and using that for the IDE integration? If so, just use "Start GHCid" in VS Code and it does all that for you.

Reply:
Oh, yes I was using the original method. I'll give the 'start' a try

Reply:
I found the Start GHCid command and it worked.  But the instructions might need an update as they still mention creating the text file.